benchmarks/dynamo/pr_time_benchmarks/benchmark_base.py

```diff
@@ -51,44 +53,68 @@
 
 
 class BenchmarkBase(ABC):
+    # measure total number of instruction spent in _work.
     _instruction_count = False
 
+    # measure total number of instruction spent in convert_frame.compile_inner
+    # TODO is there other parts we need to add ?
+    _compile_time_instruction_count = False
+
     def enable_instruction_count(self):
         self._instruction_count = True
         return self
 
+    def enable_compile_time_instruction_count(self):
+        self._compile_time_instruction_count = True
+        return self
+
     def name(self):
         return ""
 
     def description(self):
         return ""
 
     @abstractmethod
-    def prepare(self):
+    def _prepare(self):
         pass
 
     @abstractmethod
-    def work(self):
+    def _work(self):
         pass
 
-    def prepare_once(self):  # noqa: B027
+    def _prepare_once(self):  # noqa: B027
         pass
 
-    def count_instructions(self):
+    def _count_instructions(self):
         print(f"collecting instruction count for {self.name()}")
-        self.prepare_once()
```

Comment:
Do we not need this call anymore? Oh - I guess it moved to collect_all()...

Reply:
yeh once for all metrics not needed